assets/ts/schedule/components/__schedule.d.ts

```diff
@@ -120,12 +120,12 @@ export interface SimpleStopMap {
 }
 
 export interface ScheduleNote {
-  peak_service: string;
+  alternate_text: string | null;
+  exceptions: ServiceException[];
```

Comment:
Oh yeah, good catch on the order. 

And for some reason I thought you'd removed this `exceptions` from the Elixir type, though now I see I was mistaken. You _could_ remove them, since you deleted that Elixir test which was as far as I can tell the only thing that needed that value. Or you could put the test back in.

Reply:
I left it because I assume it served some purpose before and that purpose might resurface at some point.